### ISSUE TITLE
Bug with focus on input on mobile device (Android 4 or more recent)

### ISSUE THREAD

**Issue by batata004** [Semantic-Org/Semantic-UI#4094]
This bug is earily reproduceable: open the link http://bit.ly/1Xp2vvx in a smartphone (I tried at my Chrome on Android 5). As soon as you open that page you will see a modal windows gets displayed and the first input receives focus (automatically) and you can see the cursor blinking inside the input. But the smartphone's keyboard does not get displayed. The user have to really click the input in order to be able to type anything.

I think that if the cursor starts blinking inside the input the user believes he/she can start typing but that's not the case cause aparently the input has focus but the keyboard does not show up.

This bug does not happen in the browser Chrome at a notebook with WINDOWS 10, I mean: the first input gets focused as soon the page opens and the user can start typing right away without clicking the input. So I think this bug only happens on mobile platforms.

The well formated source code of this bug is attached so you can check it easier.

[index2.txt](https://github.com/Semantic-Org/Semantic-UI/files/298070/index2.txt)

**Comment by jlukic** [comment]
Currently I'm without a way to test Android, however a kindly contributor is sending me an Android phone in the next couple weeks, and I'll be revisiting any bug marked 'android'.

**Comment by batata004** [comment]
@jlukic thanks it will be really nice for you to have and Android and check this bug cause Semantic UI is not as realiable and smooth in android as it is on iphones. With and Android I think you will be able to check many other bugs and make this nice library even better!

**Comment by Captain-GaDo** [comment]
+1. This bug is very painful

**Comment by batata004** [comment]
It's sad no one more is looking trying to solve this bug. It is marked as "LOW PRIORITY" but to me, it should SEVERE PRIORITY cause in brazil at least 70% of the smartphones run Android and this bug is affecting LOTS of people which dont know what's going on cause it's not an easy bug to spot on.

**Comment by philrykoff** [comment]
In the spirit of open source, those with the biggest pain should fix it and create a PR 😉 

On the other hand, I worked around this by blurring (e.g., focussing another element) and then focussing the element again. The user doesn't notice as it's faster than meets the eye and the keyboard will show as expected.

**Comment by batata004** [comment]
@philrykoff I totally agree with you :) But it would be more practical and for the well being of the community that a fix should be provided cause many other people may be facing this very same problem and have no idea it is being caused by semantic ui.

**Comment by philrykoff** [comment]
On the other hand, maybe it's a feature of Android Chrome. 

When some pages always highlight the first textfield on any page (a nice idea for desktop browsing, because you're always ready to type), the iPhone keyboard opens on every page.

So maybe the fix for mobile should be to disable autofocus completely? Then, the textfield does not indicate focus for Android users which confuses them and it does not go on the nerves of iPhone users 😄

**Comment by batata004** [comment]
@philrykoff the problem is not the feature, the problem is that when android highlight's the first textfield I cant scroll down anymore (the scrollbars disappears forever). This bug is a little related to this https://github.com/Semantic-Org/Semantic-UI/issues/4196

**Comment by batata004** [comment]
Hey guys, any update into this? I am using semantic ui in loooots of websites and I still have to use "hacks" to fix this problem. When will this fix be commited to the mainstream?

**Comment by batata004** [comment]
Can someone mark this bug with higher priority? It affects all users with Android and Google Chrome.

**Comment by WebAsk** [comment]
Bug is generated by bootstrap framework. Remove input element from "col....." container to solve the issue.

**Comment by batata004** [comment]
"col" container @WebAsk ? I couldnt find this in my source code.

Also, if this bug is not related to semantic ui, I should not expect it to be solved? What should I do? Can any mod report it to bootstrap?

And I dont understand this: I thought semantic ui was entirely indepedent fom bootstrap. How do you say they are related?

**Comment by Banandrew** [comment]
@batata004 I think he just posted to the wrong thread on accident. The issue has nothing to do with Bootstrap.

**Comment by WebAsk** [comment]
Scuse me I wrote on wrong thread.

**Comment by batata004** [comment]
:)

**Comment by lubber-de** [comment]
This is fixed since https://github.com/fomantic/Fomantic-UI 2.7.0

**Comment by batata004** [comment]
Fomantic is doing a great job thanks to peope like @lubber-de . Anyone reading this, I suggest droping semantic and moving to fomantic which, so far, is totally compatible and way better
